README.md

```diff
@@ -111,21 +111,39 @@ Everything you see here actually works, this is not just for show!
 
 # How do I run it
 
-Currently we only support running Elsa with [Cask](https://github.com/cask/cask).
+Elsa can be run with [makem.sh](https://github.com/alphapapa/makem.sh)
+or with [Cask](https://github.com/cask/cask).
+
```

Comment:
Looks good.

There are other PRs trying to remove the dependency on Cask and honestly I'm a bit disapointed in Cask lately.  Simply because of lack of maintanance and things randomly breaking on newer Emacs. I would like to provide some vanilla bash script to run things along with the other options like emake or makem (haven't heard of it yet, will check out!).